### ISSUE TITLE
Code for Domain Generalization Experimentation

### ISSUE THREAD

**Issue by humbleOldSage** [YyzHarry/multi-domain-imbalance#6]
Which among  the three model selection methods proposed in the DomainBed paper did you use?

Does this repository has the code used for experimenting and creating Table 9,i.e., experiments related to Domain Generalization?

**Comment by YyzHarry** [comment]
Hi - thanks for your interest. As said in the paper, results for all algorithms except BoDA in the DG setting (e.g., Table 9) are directly copied from the [DomainBed paper](https://openreview.net/pdf?id=lQdXeXDoWtI) (note that the numbers in their arXiv version is not the latest; please refer to its [ICLR published version](https://openreview.net/pdf?id=lQdXeXDoWtI)). The model selection method is `training-domain validation set`.
And because of this, our repo does not contain code for DG; yet it is easy to experiment those settings using the DomainBed codebase.

**Comment by humbleOldSage** [comment]
Hi, Thanks for the help.
